spec/support/taxon_helper.rb

```diff
@@ -0,0 +1,31 @@
+module Support
+  class TaxonHelper
+    def self.expanded_link_hash(content_id, paths)
+      path_converter = lambda do |path|
+        head, *tail = path
+
+        return {} if head.nil?
+        links_hash =
+          if tail.empty?
+            {}
+          else
+            { "links" =>
+               {
+                 tail.length == 1 ? "root_taxon" : "parent_taxons" => [path_converter.call(tail)]
+               } }
```

Comment:
This formatting is rather unique, laying it out like the hash at the bottom of the file would be easier on the eyes in my opinion?